stream-testkit/src/main/scala/org/apache/pekko/stream/testkit/StreamTestKit.scala

```diff
@@ -462,6 +463,16 @@ object TestSubscriber {
       self
     }
 
+    /**
+     * Fluent DSL
+     * Expect the given elements to be signalled in order.
+     * @since 1.1.0
+     */
+    def expectNextN(elems: java.util.List[I]): Self = {
```

Comment:
any chance this could be `java.util.Iterable`?

Reply:
I think the Iterable not mean sequence/keep the order?

Reply:
https://openjdk.org/jeps/431 was only added in JDK21

Reply:
`java.util.Set` implements `Iterable` but would not keep the order. In this context a `List` would probably make most sense to guide the user.

Reply:
fair points - stick with List